src/main/resources/callCachingWorkflows/dockerHashNoLookup/dockerHashNoLookup.wdl

```diff
@@ -0,0 +1,13 @@
+task dockerhub {
+    command {
+        echo "bonjour tout le monde !"
```

Comment:
OOI is this the French "hello world"? Seems pretty close, and "hello everyone" is certainly a lot friendlier than saying hi to a lump of rock...